includes/Data/Patterns.php

```diff
@@ -130,7 +130,7 @@ public static function get_pattern_from_slug( $pattern_slug ) {
 		return false;
 	}
 
-	public static function get_theme_step_patterns_from_step( $step, $squash = false ) {
+	protected static function get_theme_step_patterns_from_step( $step, $squash = false ) {
```

Comment:
The function is also used outside this class in `PatternsController` that does not extend this class, pretty sure this is a fatal error.